src/main/java/training/subsystems/elbow/Elbow.java

```diff
@@ -0,0 +1,53 @@
+package training.subsystems.elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Elbow extends GBSubsystem {
+
+	private static CANSparkMax motor;
+	private static Elbow instance;
+
+	private Elbow() {
+		motor = new CANSparkMax(ElbowConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getPIDController().setP(ElbowConstants.MOTOR_ID_P);
+		motor.getPIDController().setI(ElbowConstants.MOTOR_ID_I);
+		motor.getPIDController().setD(ElbowConstants.MOTOR_ID_D);
+	}
+
+	public static void init() {
+		if (instance == null) {
+			instance = new Elbow();
+		}
+	}
+
+	public static Elbow getInstance() {
+		init();
+		return instance;
+	}
+
+	public static void goToAngel(Rotation2d targetAngle) {
```

Comment:
![image](https://github.com/user-attachments/assets/f3c73eb0-144c-4c4b-ad39-c17a320d8ba7)
consider changing to angle